pkg/config/config.go

```diff
@@ -63,9 +63,10 @@ type EventstoreConfig struct {
 }
 
 type AuthConfig struct {
-	Provider       string `mapstructure:"provider"`
-	PublicKey      string `mapstructure:"publicKey"`
-	UserIdentifier string `mapstructure:"userIdentifier"`
+	Provider         string `mapstructure:"provider"`
+	PublicKey        string `mapstructure:"publicKey"`
+	UserIdentifier   string `mapstructure:"userIdentifier"`
```

Comment:
Maybe we can rename these two properties to `UserIdClaim` and `TenantIdClaim` or `UserClaim` and `TenantClaim`

Reply:
Updated to `UserIdClaim` and `TenantIdClaim`

Reply:
Need to update the sample configs in the mysql and postgres READMEs as well